cpuid_arm64.c

```diff
@@ -284,6 +289,8 @@ void get_cpuconfig(void)
       printf("#define %s\n", cpuname[d]);
       // Fall-through
 		case CPU_ARMV8:
+		// case CPU_CORTEXA53;
+		// case CPU_CORTEXA55;
```

Comment:
this needs to be an actual "case CPU_CORTEXA55:" (and ending with a colon) else it will not set DTB_DEFAULT_ENTRIES for the A55 (and I think the commented-out case line for A53 should not be added, we are falling through from CORTEXA53 just a couple of lines above).
Or perhaps the "case CPU_CORTEXA55:" could be added to the A57/A72/A73 block of cases below, unless we are
certain that a "typical" A55 chipset will be as cache-limited as the older A53 designs ?

Reply:
and actually that "case CPU_CORTEX_A55:" needs to come directly after the "case CPU_CORTEXA53:" line (so that in both cases the "print #define cpuname" is executed - else you end up with wrong UNROLL parameters later)

Reply:
I tried to mark the sport that both A53 and A55 fall through, probably needs own section.

Reply:
the point is that the sequence needs to be
```
case CPU_CORTEXA53:
case CPU_CORTEXA55:
 (write the cpuname define to the config file)
case ARMV8:
(the fall-through, write what is needed by all three)
```
so no real need to mark anything (except note the fact that it is intentional fall-through and not a missing break there)

Reply:
OK, got it.